the-collab-lab/tcl-47-smart-shopping-list
# WN-TF-alerts

## Description

This code modifies the AddItem.jsx file to include an MUI Alert component. The alert component provides a better user experience than the previous inline notification with appropriate color coding, depending on the nature of the message.

## Related Issue

Issue #33  (this is part one of 2, so issue should remain open)

## Acceptance Criteria

- [x] Add MUI Alert Component to our List component.
- [x]  Look through our existing logic and make all alerts/notifications are being routed to the new Alert Component
- [x] Ensure that each alert uses an appropriate alert type.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :sparkles: New feature     |

## Updates

AddItem.jsx

### Before
![before](https://user-images.githubusercontent.com/94068349/204907022-334229cb-36a1-4741-91cd-3dc4dbdf8ae2.png)

### After

![after1](https://user-images.githubusercontent.com/94068349/204907065-2e38f28e-5ae7-4ace-93f4-77b23ebde9c8.png)
![after2](https://user-images.githubusercontent.com/94068349/204907079-d57d7ee0-e813-4d6e-b132-d4f83548b012.png)
![after3](https://user-images.githubusercontent.com/94068349/204907093-1496126d-07a5-4031-bb96-720fe9d5965a.png)


## Testing Steps / QA Criteria

- git pull WN-TF-alerts
- npm start
- add a new item to the list to check for success alert
- add a blank item to the list to check for warning alert
- add a duplicate item to the list to check for warning alert
